vault/logical_passthrough.go

```diff
@@ -105,14 +130,21 @@ func (b *PassthroughBackend) handleRead(
 		ttl, _ = rawData["ttl"].(string)
 	}
 
+	var ttlDuration time.Duration
```

Comment:
May as well directly do `ttlDuration := b.System().DefaultLeaseTTL()` and save an extra branch